lib/active_record_florder.rb

```diff
@@ -55,6 +55,14 @@ def step(value)
     def get_step
       @step || 2**16
     end
+
+    def populate(value)
+      @populate = value
+    end
+
+    def get_populate
```

Comment:
:+1: this is next thing I want to try. And next thing I'm not much happy about is: https://github.com/turboMaCk/active-record-florder/blob/master/lib/active_record_florder/configurable.rb